api/services/managedInstance/aws/paths/managedID.yaml

```diff
@@ -16,6 +16,12 @@ put:
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
     - $ref: "../parameters/mi-id.yaml"
+  requestBody:
+    content:
+      application/json:
+        schema:
+          allOf:
+            - $ref: "../schemas/mi-update.yaml"
```

Comment:
@vlad666spot why not using the regular mi schema?

Reply:
The same like in previous issue - in create API call some params are required - but in update API Call the same params are not required)